4-projects/modules/standard_projects/main.tf

```diff
@@ -64,6 +67,27 @@ module "prod_project" {
   }
 }
 
+module "dev_project" {
+  source                      = "terraform-google-modules/project-factory/google"
+  version                     = "~> 7.0"
+  random_project_id           = "true"
+  impersonate_service_account = var.impersonate_service_account
+  activate_apis               = var.activate_apis
+  name                        = "${var.project_prefix}-dev"
+  org_id                      = var.org_id
+  billing_account             = var.billing_account
+  folder_id                   = var.dev_folder_id
+
```

Comment:
expose `skip_gcloud_download ` here and though out with default to `true`

Reply:
+1 - if we can do this for all projects that would be great